docs/ref-conflict-server.md

```diff
@@ -51,67 +51,25 @@ Every conflict can be handled using a set of predefined steps
     // 2. Check for conflicts
-    if (conflictHandler.hasConflict(serverData,clientData)) {
-      // 3. Resolve conflict (client or server) and return error to client
-      return await conflictHandler.resolveOnClient(serverData, clientData).response
-    }
-    // 4. Call next state to update
-    greeting = conflictHandler.nextState(clientData)
+    conflictHandler.checkForConflicts(serverData,clientData)
     // 5. Save object to data source
 ```
 
-Resolvers can be implemented to handle conflicts on client or server.
-Depending on  the strategy used, the resolver implementation will differ.
-Please see the chapter below for individual implementations.
-
-## Options for Resolving Conflicts
-
-Conflicts can be resolved on server or client depending on the resolver implementation
-
-### Conflicts on the Client
-
-```javascript
-// Data
-const serverState = ...
-
-changeGreeting: async (obj, clientState, context, info) => {
-    if (conflictHandler.hasConflict(serverState, args)) {
-      const clientState = args
-      return await conflictHandler.resolveOnClient(serverState, clientState).response
-    }
-    serverState = conflictHandler.nextState(clientState)
-    return serverState
-}
-```
-
-### Conflicts on the Server
-
-```javascript
-// Data
-const serverState = ...
-
- changeGreeting: async (obj, clientState, context, info) => {
-      if (conflictHandler.hasConflict(serverState, clientState)) {
-        const strategy = customGreetingResolutionStrategy
-        const { resolvedState, response } = await conflictHandler.resolveOnServer(strategy, serverState, clientState)
-        serverState = resolvedState
-        return response
-      }
-      serverState = conflictHandler.nextState(clientState)
-      return serverState
-    }
-```
-
-> Note: For complete implementation see example application located in `examples/conflicts` folder.
+Resolvers can be implemented to handle conflicts on client .
 
 ## Implementing Custom Conflict Mechanism
 
-The`ObjectState` interface is a complete conflict resolution implementation that provides a set of rules to detect and handle conflict. Interface will allow developers to handle conflict on the client or the server. `nextSate` method is a way for interface to modify existing object before is being saved to the database.
+The`ObjectState` interface is a complete conflict resolution implementation that provides a set of rules to detect and handle conflict. Interface will allow developers to handle conflict on the client. Client side application will need to match the server side implementation 
 For example when using `lastModified` field as a way to detect conflicts:
 
 ```typescript
- public nextState(currentObjectState: ObjectStateData) {
-    currentObjectState.lastModified = new Date()
-    return currentObjectState
+  public checkForConflict(serverState: ObjectStateData, clientState: ObjectStateData) {
+    if(serverState.lastModified === clientState.lastModified){
```

Comment:
I might be missing something here but this looks funny to me. This is an example of custom conflict resolution but would it not make sense to be throwing the `ObjectConflictError` when those values are not equal? i.e. `if(serverState.lastModified !== clientState.lastModified)` ??